wdl-format/src/format/format_state.rs

```diff
@@ -0,0 +1,79 @@
+//! Contains the `FormatState` struct, which is used to keep track of the
+//! current formatting state. This includes the current indentation level and
+//! whether the current line has been interrupted by comments.
+//! The state becomes "interrupted" by comments when a comment forces a newline
+//! where it would otherwise not be expected. In this case, the next line(s)
+//! will be indented by one level.
+use std::fmt::Write;
+
+use anyhow::Result;
+
+/// Space constant used for formatting.
+pub const SPACE: &str = " ";
+/// Indentation constant used for formatting.
+pub const INDENT: &str = "    ";
```

Comment:
For readability, can you do something like this?

```suggestion
pub const INDENT: &str = SPACE.repeat(4);
```

Reply:
Could use some input from Clay or Peter on this. I messed around with this for a few minutes and was getting various compiler errors. I _think_ the right way to do this is using a `lazy_static`?

Reply:
@claymcleod / @peterhuene